Description: Use system LDB headers rather than the headers of the bundled and
 unused LDB.
Author: Jelmer Vernooij <jelmer@samba.org>

=== modified file 'lib/util/util_ldb.c'
--- old/lib/util/util_ldb.c	2010-10-15 01:32:48 +0000
+++ new/lib/util/util_ldb.c	2011-02-27 16:49:39 +0000
@@ -21,7 +21,7 @@
 */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "../lib/util/util_ldb.h"
 
 /*

=== modified file 'libcli/ldap/ldap_message.h'
--- old/libcli/ldap/ldap_message.h	2010-10-15 01:32:48 +0000
+++ new/libcli/ldap/ldap_message.h	2011-02-27 16:48:59 +0000
@@ -25,7 +25,7 @@
 #if _SAMBA_BUILD_ == 3
 #include "lib/ldb_compat.h"
 #else
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #endif
 
 enum ldap_request_tag {

=== modified file 'libcli/ldap/ldap_ndr.c'
--- old/libcli/ldap/ldap_ndr.c	2011-01-25 09:22:16 +0000
+++ new/libcli/ldap/ldap_ndr.c	2011-02-27 16:49:37 +0000
@@ -24,7 +24,7 @@
 #if _SAMBA_BUILD_ == 3
 #include "lib/ldb_compat.h"
 #else
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #endif
 #include "librpc/gen_ndr/ndr_security.h"
 #include "librpc/gen_ndr/ndr_misc.h"

=== modified file 'source4/auth/credentials/credentials_secrets.c'
--- old/source4/auth/credentials/credentials_secrets.c	2010-10-15 01:32:48 +0000
+++ new/source4/auth/credentials/credentials_secrets.c	2011-02-27 16:49:42 +0000
@@ -23,7 +23,7 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "librpc/gen_ndr/samr.h" /* for struct samrPassword */
 #include "param/secrets.h"
 #include "system/filesys.h"

=== modified file 'source4/auth/gensec/gensec_gssapi.c'
--- old/source4/auth/gensec/gensec_gssapi.c	2011-01-25 09:22:16 +0000
+++ new/source4/auth/gensec/gensec_gssapi.c	2011-02-27 16:49:45 +0000
@@ -27,7 +27,7 @@
 #include "auth/kerberos/kerberos.h"
 #include "librpc/gen_ndr/krb5pac.h"
 #include "auth/auth.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/auth_sam.h"
 #include "librpc/rpc/dcerpc.h"
 #include "auth/credentials/credentials.h"

=== modified file 'source4/auth/kerberos/kerberos_pac.c'
--- old/source4/auth/kerberos/kerberos_pac.c	2010-10-15 01:32:48 +0000
+++ new/source4/auth/kerberos/kerberos_pac.c	2011-02-27 16:49:49 +0000
@@ -28,7 +28,7 @@
 #include "auth/auth.h"
 #include "auth/kerberos/kerberos.h"
 #include "librpc/gen_ndr/ndr_krb5pac.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/auth_sam_reply.h"
 
 krb5_error_code check_pac_checksum(TALLOC_CTX *mem_ctx,

=== modified file 'source4/auth/ntlm/auth_sam.c'
--- old/source4/auth/ntlm/auth_sam.c	2011-01-25 09:22:16 +0000
+++ new/source4/auth/ntlm/auth_sam.c	2011-02-27 16:49:51 +0000
@@ -21,7 +21,7 @@
 
 #include "includes.h"
 #include "system/time.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "libcli/ldap/ldap_ndr.h"
 #include "libcli/security/security.h"
 #include "auth/auth.h"

=== modified file 'source4/cldap_server/cldap_server.c'
--- old/source4/cldap_server/cldap_server.c	2010-10-15 01:32:48 +0000
+++ new/source4/cldap_server/cldap_server.c	2011-02-27 16:49:58 +0000
@@ -27,8 +27,8 @@
 #include "cldap_server/cldap_server.h"
 #include "system/network.h"
 #include "lib/socket/netif.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "dsdb/samdb/samdb.h"
 #include "ldb_wrap.h"
 #include "auth/auth.h"

=== modified file 'source4/cldap_server/netlogon.c'
--- old/source4/cldap_server/netlogon.c	2011-01-25 09:22:16 +0000
+++ new/source4/cldap_server/netlogon.c	2011-02-27 16:50:06 +0000
@@ -21,8 +21,8 @@
 */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "lib/events/events.h"
 #include "smbd/service_task.h"
 #include "cldap_server/cldap_server.h"

=== modified file 'source4/cldap_server/rootdse.c'
--- old/source4/cldap_server/rootdse.c	2010-09-08 03:30:03 +0000
+++ new/source4/cldap_server/rootdse.c	2011-02-27 16:50:12 +0000
@@ -21,8 +21,8 @@
 
 #include "includes.h"
 #include <tevent.h>
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "smbd/service_task.h"
 #include "cldap_server/cldap_server.h"
 #include "librpc/gen_ndr/ndr_misc.h"

=== modified file 'source4/dsdb/common/dsdb_dn.c'
--- old/source4/dsdb/common/dsdb_dn.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/common/dsdb_dn.c	2011-02-27 16:50:15 +0000
@@ -21,7 +21,7 @@
 
 #include "includes.h"
 #include "dsdb/samdb/samdb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "librpc/ndr/libndr.h"
 #include "libcli/security/dom_sid.h"
 

=== modified file 'source4/dsdb/common/tests/dsdb_dn.c'
--- old/source4/dsdb/common/tests/dsdb_dn.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/common/tests/dsdb_dn.c	2011-02-27 16:50:25 +0000
@@ -21,8 +21,8 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "lib/ldb-samba/ldif_handlers.h"
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"

=== modified file 'source4/dsdb/kcc/kcc_connection.c'
--- old/source4/dsdb/kcc/kcc_connection.c	2010-09-28 09:46:54 +0000
+++ new/source4/dsdb/kcc/kcc_connection.c	2011-02-27 16:50:28 +0000
@@ -27,7 +27,7 @@
 #include "lib/messaging/irpc.h"
 #include "dsdb/kcc/kcc_service.h"
 #include "dsdb/kcc/kcc_connection.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/kcc/kcc_deleted.c'
--- old/source4/dsdb/kcc/kcc_deleted.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/kcc/kcc_deleted.c	2011-02-27 16:50:33 +0000
@@ -28,7 +28,7 @@
 #include "lib/messaging/irpc.h"
 #include "dsdb/kcc/kcc_connection.h"
 #include "dsdb/kcc/kcc_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/kcc/kcc_drs_replica_info.c'
--- old/source4/dsdb/kcc/kcc_drs_replica_info.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/kcc/kcc_drs_replica_info.c	2011-02-27 16:50:39 +0000
@@ -28,7 +28,7 @@
 #include "lib/events/events.h"
 #include "lib/messaging/irpc.h"
 #include "dsdb/kcc/kcc_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/kcc/kcc_periodic.c'
--- old/source4/dsdb/kcc/kcc_periodic.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/kcc/kcc_periodic.c	2011-02-27 16:50:42 +0000
@@ -28,7 +28,7 @@
 #include "lib/messaging/irpc.h"
 #include "dsdb/kcc/kcc_connection.h"
 #include "dsdb/kcc/kcc_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/kcc/kcc_service.c'
--- old/source4/dsdb/kcc/kcc_service.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/kcc/kcc_service.c	2011-02-27 16:50:47 +0000
@@ -28,7 +28,7 @@
 #include "lib/events/events.h"
 #include "lib/messaging/irpc.h"
 #include "dsdb/kcc/kcc_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_notify.c'
--- old/source4/dsdb/repl/drepl_notify.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/drepl_notify.c	2011-02-27 16:50:52 +0000
@@ -27,7 +27,7 @@
 #include "auth/auth.h"
 #include "smbd/service.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_out_helpers.c'
--- old/source4/dsdb/repl/drepl_out_helpers.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/drepl_out_helpers.c	2011-02-27 16:50:55 +0000
@@ -25,7 +25,7 @@
 #include "smbd/service.h"
 #include "lib/events/events.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_out_pull.c'
--- old/source4/dsdb/repl/drepl_out_pull.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/drepl_out_pull.c	2011-02-27 16:50:58 +0000
@@ -25,7 +25,7 @@
 #include "smbd/service.h"
 #include "lib/events/events.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_partitions.c'
--- old/source4/dsdb/repl/drepl_partitions.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/drepl_partitions.c	2011-02-27 16:51:02 +0000
@@ -25,7 +25,7 @@
 #include "smbd/service.h"
 #include "lib/events/events.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_periodic.c'
--- old/source4/dsdb/repl/drepl_periodic.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/repl/drepl_periodic.c	2011-02-27 16:51:06 +0000
@@ -25,7 +25,7 @@
 #include "auth/auth.h"
 #include "smbd/service.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/drepl_service.c'
--- old/source4/dsdb/repl/drepl_service.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/drepl_service.c	2011-02-27 16:51:12 +0000
@@ -26,7 +26,7 @@
 #include "smbd/service.h"
 #include "lib/events/events.h"
 #include "dsdb/repl/drepl_service.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/repl/replicated_objects.c'
--- old/source4/dsdb/repl/replicated_objects.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/repl/replicated_objects.c	2011-02-27 16:51:15 +0000
@@ -21,7 +21,7 @@
 
 #include "includes.h"
 #include "dsdb/samdb/samdb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"

=== modified file 'source4/dsdb/samdb/cracknames.c'
--- old/source4/dsdb/samdb/cracknames.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/cracknames.c	2011-02-27 16:51:23 +0000
@@ -25,8 +25,8 @@
 #include "librpc/gen_ndr/drsuapi.h"
 #include "lib/events/events.h"
 #include "rpc_server/common/common.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "system/kerberos.h"
 #include "auth/kerberos/kerberos.h"
 #include "libcli/ldap/ldap_ndr.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/extended_dn_in.c'
--- old/source4/dsdb/samdb/ldb_modules/extended_dn_in.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/extended_dn_in.c	2011-02-27 16:51:33 +0000
@@ -30,9 +30,9 @@
  */
 
 #include "includes.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
-#include "ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 
 /*
   TODO: if relax is not set then we need to reject the fancy RMD_* and

=== modified file 'source4/dsdb/samdb/ldb_modules/extended_dn_out.c'
--- old/source4/dsdb/samdb/ldb_modules/extended_dn_out.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/extended_dn_out.c	2011-02-27 16:51:43 +0000
@@ -32,9 +32,9 @@
  */
 
 #include "includes.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
-#include "ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "libcli/security/security.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_security.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/extended_dn_store.c'
--- old/source4/dsdb/samdb/ldb_modules/extended_dn_store.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/extended_dn_store.c	2011-02-27 16:51:56 +0000
@@ -36,9 +36,9 @@
  */
 
 #include "includes.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
-#include "ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "dsdb/samdb/samdb.h"
 #include "libcli/security/security.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/partition.h'
--- old/source4/dsdb/samdb/ldb_modules/partition.h	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/partition.h	2011-02-27 16:48:48 +0000
@@ -18,9 +18,9 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/samdb/ldb_modules/util.h"
 #include "system/locale.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/rootdse.c'
--- old/source4/dsdb/samdb/ldb_modules/rootdse.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/rootdse.c	2011-02-27 16:52:05 +0000
@@ -21,8 +21,8 @@
 */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_module.h>
 #include "system/time.h"
 #include "dsdb/samdb/samdb.h"
 #include "version.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/samba_dsdb.c'
--- old/source4/dsdb/samdb/ldb_modules/samba_dsdb.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/samba_dsdb.c	2011-02-27 16:52:16 +0000
@@ -31,9 +31,9 @@
  */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "dsdb/samdb/ldb_modules/util.h"
 #include "dsdb/samdb/samdb.h"
 #include "librpc/ndr/libndr.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/samba_secrets.c'
--- old/source4/dsdb/samdb/ldb_modules/samba_secrets.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/samba_secrets.c	2011-02-27 16:52:45 +0000
@@ -30,9 +30,9 @@
  */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "dsdb/samdb/ldb_modules/util.h"
 #include "dsdb/samdb/samdb.h"
 

=== modified file 'source4/dsdb/samdb/ldb_modules/schema_util.c'
--- old/source4/dsdb/samdb/ldb_modules/schema_util.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/samdb/ldb_modules/schema_util.c	2011-02-27 16:52:48 +0000
@@ -25,7 +25,7 @@
 #include "dsdb/common/util.h"
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/samdb/ldb_modules/util.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "librpc/gen_ndr/ndr_drsuapi.h"
 #include "librpc/gen_ndr/ndr_drsblobs.h"
 #include "param/param.h"

=== modified file 'source4/dsdb/samdb/ldb_modules/show_deleted.c'
--- old/source4/dsdb/samdb/ldb_modules/show_deleted.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/show_deleted.c	2011-02-27 16:52:52 +0000
@@ -32,7 +32,7 @@
  */
 
 #include "includes.h"
-#include "ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/samdb/ldb_modules/util.h"
 

=== modified file 'source4/dsdb/samdb/ldb_modules/simple_ldap_map.c'
--- old/source4/dsdb/samdb/ldb_modules/simple_ldap_map.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/ldb_modules/simple_ldap_map.c	2011-02-27 16:53:04 +0000
@@ -27,7 +27,7 @@
 */
 
 #include "includes.h"
-#include "ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "ldb/ldb_map/ldb_map.h"
 
 #include "librpc/gen_ndr/ndr_misc.h"

=== modified file 'source4/dsdb/samdb/samdb.c'
--- old/source4/dsdb/samdb/samdb.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/samdb.c	2011-02-27 16:53:12 +0000
@@ -26,8 +26,8 @@
 #include "librpc/gen_ndr/ndr_security.h"
 #include "lib/events/events.h"
 #include "lib/ldb-samba/ldb_wrap.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "libcli/security/security.h"
 #include "libcli/auth/libcli_auth.h"
 #include "libcli/ldap/ldap_ndr.h"

=== modified file 'source4/dsdb/samdb/samdb.h'
--- old/source4/dsdb/samdb/samdb.h	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/samdb/samdb.h	2011-02-27 16:49:05 +0000
@@ -30,7 +30,7 @@
 struct tevent_context;
 
 #include "librpc/gen_ndr/security.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "lib/ldb-samba/ldif_handlers.h"
 #include "librpc/gen_ndr/samr.h"
 #include "librpc/gen_ndr/drsuapi.h"

=== modified file 'source4/dsdb/schema/dsdb_dn.c'
--- old/source4/dsdb/schema/dsdb_dn.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/schema/dsdb_dn.c	2011-02-27 16:53:18 +0000
@@ -21,7 +21,7 @@
 
 #include "includes.h"
 #include "dsdb/samdb/samdb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "librpc/ndr/libndr.h"
 #include "libcli/security/dom_sid.h"
 

=== modified file 'source4/dsdb/schema/schema_filtered.c'
--- old/source4/dsdb/schema/schema_filtered.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/schema/schema_filtered.c	2011-02-27 16:53:22 +0000
@@ -21,7 +21,7 @@
 #include "includes.h"
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/common/util.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "param/param.h"
 

=== modified file 'source4/dsdb/schema/schema_info_attr.c'
--- old/source4/dsdb/schema/schema_info_attr.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/schema/schema_info_attr.c	2011-02-27 16:53:25 +0000
@@ -23,7 +23,7 @@
 #include "dsdb/common/util.h"
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/samdb/ldb_modules/util.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "librpc/gen_ndr/ndr_drsuapi.h"
 #include "librpc/gen_ndr/ndr_drsblobs.h"
 #include "param/param.h"

=== modified file 'source4/dsdb/schema/schema_init.c'
--- old/source4/dsdb/schema/schema_init.c	2011-01-25 09:22:16 +0000
+++ new/source4/dsdb/schema/schema_init.c	2011-02-27 16:53:32 +0000
@@ -23,13 +23,13 @@
 #include "includes.h"
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/common/util.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "../lib/util/dlinklist.h"
 #include "librpc/gen_ndr/ndr_misc.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"
 #include "librpc/gen_ndr/ndr_drsblobs.h"
 #include "param/param.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "../lib/util/asn1.h"
 
 

=== modified file 'source4/dsdb/schema/schema_set.c'
--- old/source4/dsdb/schema/schema_set.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/schema/schema_set.c	2011-02-27 16:53:37 +0000
@@ -23,7 +23,7 @@
 #include "includes.h"
 #include "lib/util/dlinklist.h"
 #include "dsdb/samdb/samdb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "param/param.h"
 #include "librpc/ndr/libndr.h"
 #include "librpc/gen_ndr/ndr_misc.h"

=== modified file 'source4/dsdb/schema/schema_syntax.c'
--- old/source4/dsdb/schema/schema_syntax.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/schema/schema_syntax.c	2011-02-27 16:53:43 +0000
@@ -25,8 +25,8 @@
 #include "librpc/gen_ndr/ndr_drsuapi.h"
 #include "librpc/gen_ndr/ndr_security.h"
 #include "librpc/gen_ndr/ndr_misc.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "system/time.h"
 #include "../lib/util/charset/charset.h"
 #include "librpc/ndr/libndr.h"

=== modified file 'source4/dsdb/schema/tests/schema_syntax.c'
--- old/source4/dsdb/schema/tests/schema_syntax.c	2010-10-15 01:32:48 +0000
+++ new/source4/dsdb/schema/tests/schema_syntax.c	2011-02-27 16:53:55 +0000
@@ -21,8 +21,8 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "lib/ldb-samba/ldif_handlers.h"
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"

=== modified file 'source4/kdc/pac-glue.c'
--- old/source4/kdc/pac-glue.c	2010-10-15 01:32:48 +0000
+++ new/source4/kdc/pac-glue.c	2011-02-27 16:53:59 +0000
@@ -23,7 +23,7 @@
 
 #include "includes.h"
 #include "../libds/common/flags.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/auth.h"
 #include "auth/auth_sam_reply.h"
 #include "kdc/kdc-glue.h"

=== modified file 'source4/ldap_server/ldap_backend.c'
--- old/source4/ldap_server/ldap_backend.c	2011-01-25 09:22:16 +0000
+++ new/source4/ldap_server/ldap_backend.c	2011-02-27 16:54:08 +0000
@@ -26,8 +26,8 @@
 #include "param/param.h"
 #include "smbd/service_stream.h"
 #include "dsdb/samdb/samdb.h"
-#include "lib/ldb/include/ldb_errors.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_errors.h>
+#include <ldb_module.h>
 #include "ldb_wrap.h"
 
 #define VALID_DN_SYNTAX(dn) do {\

=== modified file 'source4/ldap_server/ldap_bind.c'
--- old/source4/ldap_server/ldap_bind.c	2010-10-15 01:32:48 +0000
+++ new/source4/ldap_server/ldap_bind.c	2011-02-27 16:54:14 +0000
@@ -21,8 +21,8 @@
 #include "ldap_server/ldap_server.h"
 #include "auth/auth.h"
 #include "smbd/service.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "dsdb/samdb/samdb.h"
 #include "auth/gensec/gensec.h"
 #include "auth/gensec/gensec_tstream.h"

=== modified file 'source4/ldap_server/ldap_server.c'
--- old/source4/ldap_server/ldap_server.c	2010-10-15 01:32:48 +0000
+++ new/source4/ldap_server/ldap_server.c	2011-02-27 16:54:19 +0000
@@ -36,8 +36,8 @@
 #include "smbd/process_model.h"
 #include "lib/tls/tls.h"
 #include "lib/messaging/irpc.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "libcli/ldap/ldap_proto.h"
 #include "system/network.h"
 #include "lib/socket/netif.h"

=== modified file 'source4/lib/ldb-samba/ldb_wrap.c'
--- old/source4/lib/ldb-samba/ldb_wrap.c	2011-01-25 09:22:16 +0000
+++ new/source4/lib/ldb-samba/ldb_wrap.c	2011-02-27 16:54:37 +0000
@@ -28,8 +28,8 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "lib/ldb-samba/ldif_handlers.h"
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"

=== modified file 'source4/lib/ldb-samba/ldif_handlers.c'
--- old/source4/lib/ldb-samba/ldif_handlers.c	2011-01-25 09:22:16 +0000
+++ new/source4/lib/ldb-samba/ldif_handlers.c	2011-02-27 16:54:45 +0000
@@ -23,8 +23,8 @@
 */
 
 #include "includes.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb.h>
+#include <ldb_module.h>
 #include "ldb_handlers.h"
 #include "dsdb/samdb/samdb.h"
 #include "librpc/gen_ndr/ndr_security.h"

=== modified file 'source4/lib/policy/gp_ldap.c'
--- old/source4/lib/policy/gp_ldap.c	2011-01-25 09:22:16 +0000
+++ new/source4/lib/policy/gp_ldap.c	2011-02-27 16:55:58 +0000
@@ -19,7 +19,7 @@
  */
 #include "includes.h"
 #include "param/param.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "lib/ldb-samba/ldb_wrap.h"
 #include "auth/credentials/credentials.h"
 #include "../librpc/gen_ndr/nbt.h"

=== modified file 'source4/lib/registry/ldb.c'
--- old/source4/lib/registry/ldb.c	2011-01-25 09:22:16 +0000
+++ new/source4/lib/registry/ldb.c	2011-02-27 16:56:18 +0000
@@ -20,8 +20,8 @@
 
 #include "includes.h"
 #include "registry.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "ldb_wrap.h"
 #include "librpc/gen_ndr/winreg.h"
 #include "param/param.h"

=== modified file 'source4/libnet/libnet.h'
--- old/source4/libnet/libnet.h	2011-01-25 09:22:16 +0000
+++ new/source4/libnet/libnet.h	2011-02-27 16:49:18 +0000
@@ -60,7 +60,7 @@
 };
 
 
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "libnet/composite.h"
 #include "libnet/userman.h"
 #include "libnet/userinfo.h"

=== modified file 'source4/libnet/libnet_become_dc.c'
--- old/source4/libnet/libnet_become_dc.c	2010-10-15 01:32:48 +0000
+++ new/source4/libnet/libnet_become_dc.c	2011-02-27 16:55:21 +0000
@@ -21,8 +21,8 @@
 #include "libnet/libnet.h"
 #include "libcli/composite/composite.h"
 #include "libcli/cldap/cldap.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"
 #include "../libds/common/flags.h"

=== modified file 'source4/libnet/libnet_join.c'
--- old/source4/libnet/libnet_join.c	2011-01-25 09:22:16 +0000
+++ new/source4/libnet/libnet_join.c	2011-02-27 16:55:28 +0000
@@ -22,8 +22,8 @@
 #include "includes.h"
 #include "libnet/libnet.h"
 #include "librpc/gen_ndr/ndr_drsuapi_c.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "dsdb/samdb/samdb.h"
 #include "ldb_wrap.h"
 #include "libcli/security/security.h"

=== modified file 'source4/libnet/libnet_site.c'
--- old/source4/libnet/libnet_site.c	2010-09-08 03:30:03 +0000
+++ new/source4/libnet/libnet_site.c	2011-02-27 16:55:38 +0000
@@ -20,8 +20,8 @@
 #include "includes.h"
 #include "libnet/libnet.h"
 #include "libcli/cldap/cldap.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "libcli/resolve/resolve.h"
 #include "param/param.h"
 #include "lib/tsocket/tsocket.h"

=== modified file 'source4/libnet/libnet_unbecome_dc.c'
--- old/source4/libnet/libnet_unbecome_dc.c	2010-10-15 01:32:48 +0000
+++ new/source4/libnet/libnet_unbecome_dc.c	2011-02-27 16:55:47 +0000
@@ -21,8 +21,8 @@
 #include "libnet/libnet.h"
 #include "libcli/composite/composite.h"
 #include "libcli/cldap/cldap.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"
 #include "../libds/common/flags.h"

=== modified file 'source4/libnet/libnet_vampire.c'
--- old/source4/libnet/libnet_vampire.c	2011-01-25 09:22:16 +0000
+++ new/source4/libnet/libnet_vampire.c	2011-02-27 16:55:53 +0000
@@ -27,8 +27,8 @@
 #include "lib/events/events.h"
 #include "dsdb/samdb/samdb.h"
 #include "../lib/util/dlinklist.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "librpc/ndr/libndr.h"
 #include "librpc/gen_ndr/ndr_drsuapi.h"
 #include "librpc/gen_ndr/ndr_drsblobs.h"

=== modified file 'source4/nbt_server/dgram/netlogon.c'
--- old/source4/nbt_server/dgram/netlogon.c	2010-10-15 01:32:48 +0000
+++ new/source4/nbt_server/dgram/netlogon.c	2011-02-27 16:56:23 +0000
@@ -23,7 +23,7 @@
 #include "includes.h"
 #include "nbt_server/nbt_server.h"
 #include "lib/socket/socket.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "dsdb/samdb/samdb.h"
 #include "auth/auth.h"
 #include "param/param.h"

=== modified file 'source4/nbt_server/wins/wins_ldb.c'
--- old/source4/nbt_server/wins/wins_ldb.c	2010-10-15 01:32:48 +0000
+++ new/source4/nbt_server/wins/wins_ldb.c	2011-02-27 16:56:51 +0000
@@ -31,7 +31,7 @@
 #include "lib/events/events.h"
 #include "nbt_server/nbt_server.h"
 #include "nbt_server/wins/winsdb.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "system/network.h"
 #include "lib/socket/netif.h"
 #include "param/param.h"

=== modified file 'source4/nbt_server/wins/winsdb.c'
--- old/source4/nbt_server/wins/winsdb.c	2010-10-15 01:32:48 +0000
+++ new/source4/nbt_server/wins/winsdb.c	2011-02-27 16:56:47 +0000
@@ -23,8 +23,8 @@
 #include "includes.h"
 #include "nbt_server/nbt_server.h"
 #include "nbt_server/wins/winsdb.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "librpc/gen_ndr/ndr_nbt.h"
 #include "system/time.h"
 #include "ldb_wrap.h"

=== modified file 'source4/nbt_server/wins/winsserver.c'
--- old/source4/nbt_server/wins/winsserver.c	2010-09-08 03:30:03 +0000
+++ new/source4/nbt_server/wins/winsserver.c	2011-02-27 16:56:58 +0000
@@ -32,7 +32,7 @@
 #include "system/network.h"
 #include "lib/socket/socket.h"
 #include "lib/socket/netif.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "param/param.h"
 #include "libcli/resolve/resolve.h"
 #include "lib/util/util_net.h"

=== modified file 'source4/ntp_signd/ntp_signd.c'
--- old/source4/ntp_signd/ntp_signd.c	2011-01-25 09:22:16 +0000
+++ new/source4/ntp_signd/ntp_signd.c	2011-02-27 16:57:06 +0000
@@ -35,8 +35,8 @@
 #include "auth/auth.h"
 #include "libcli/security/security.h"
 #include "libcli/ldap/ldap_ndr.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "../lib/crypto/md5.h"
 #include "system/network.h"
 #include "system/passwd.h"

=== modified file 'source4/ntptr/simple_ldb/ntptr_simple_ldb.c'
--- old/source4/ntptr/simple_ldb/ntptr_simple_ldb.c	2010-10-15 01:32:48 +0000
+++ new/source4/ntptr/simple_ldb/ntptr_simple_ldb.c	2011-02-27 16:57:09 +0000
@@ -30,7 +30,7 @@
 #include "includes.h"
 #include "ntptr/ntptr.h"
 #include "librpc/gen_ndr/ndr_spoolss.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/auth.h"
 #include "dsdb/samdb/samdb.h"
 #include "ldb_wrap.h"

=== modified file 'source4/param/secrets.c'
--- old/source4/param/secrets.c	2011-01-25 09:22:16 +0000
+++ new/source4/param/secrets.c	2011-02-27 16:57:12 +0000
@@ -27,7 +27,7 @@
 #include "system/filesys.h"
 #include "tdb_wrap.h"
 #include "lib/ldb-samba/ldb_wrap.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "../lib/util/util_tdb.h"
 #include "librpc/gen_ndr/ndr_security.h"
 #include "dsdb/samdb/samdb.h"

=== modified file 'source4/param/share_ldb.c'
--- old/source4/param/share_ldb.c	2010-09-08 03:30:03 +0000
+++ new/source4/param/share_ldb.c	2011-02-27 16:57:19 +0000
@@ -20,8 +20,8 @@
 */
 
 #include "includes.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "auth/auth.h"
 #include "ldb_wrap.h"
 #include "param/share.h"

=== modified file 'source4/rpc_server/lsa/lsa.h'
--- old/source4/rpc_server/lsa/lsa.h	2010-09-08 03:30:03 +0000
+++ new/source4/rpc_server/lsa/lsa.h	2011-02-27 16:49:23 +0000
@@ -26,7 +26,7 @@
 #include "auth/auth.h"
 #include "dsdb/samdb/samdb.h"
 #include "libcli/ldap/ldap_ndr.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb_errors.h>
 #include "libcli/security/security.h"
 #include "libcli/auth/libcli_auth.h"
 #include "param/secrets.h"

=== modified file 'source4/rpc_server/samr/dcesrv_samr.c'
--- old/source4/rpc_server/samr/dcesrv_samr.c	2011-01-25 09:22:16 +0000
+++ new/source4/rpc_server/samr/dcesrv_samr.c	2011-02-27 16:57:26 +0000
@@ -28,8 +28,8 @@
 #include "rpc_server/common/common.h"
 #include "rpc_server/samr/dcesrv_samr.h"
 #include "system/time.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "../libds/common/flags.h"
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/common/util.h"

=== modified file 'source4/samba_tool/gpo.c'
--- old/source4/samba_tool/gpo.c	2010-10-15 01:32:48 +0000
+++ new/source4/samba_tool/gpo.c	2011-02-27 16:57:29 +0000
@@ -23,7 +23,7 @@
 
 #include "includes.h"
 #include "samba_tool/samba_tool.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/auth.h"
 #include "param/param.h"
 #include "libcli/security/sddl.h"

=== modified file 'source4/samba_tool/samba_tool.c'
--- old/source4/samba_tool/samba_tool.c	2010-10-15 01:32:48 +0000
+++ new/source4/samba_tool/samba_tool.c	2011-02-27 16:57:32 +0000
@@ -45,7 +45,7 @@
 #include "includes.h"
 #include "samba_tool/samba_tool.h"
 #include "lib/cmdline/popt_common.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "librpc/rpc/dcerpc.h"
 #include "param/param.h"
 #include "lib/events/events.h"

=== modified file 'source4/torture/drs/unit/schemainfo_tests.c'
--- old/source4/torture/drs/unit/schemainfo_tests.c	2010-10-15 01:32:48 +0000
+++ new/source4/torture/drs/unit/schemainfo_tests.c	2011-02-27 16:57:36 +0000
@@ -25,7 +25,7 @@
 #include "dsdb/samdb/samdb.h"
 #include "dsdb/samdb/ldb_modules/util.h"
 #include "ldb_wrap.h"
-#include "lib/ldb/include/ldb_module.h"
+#include <ldb_module.h>
 #include "torture/rpc/drsuapi.h"
 #include "librpc/ndr/libndr.h"
 #include "param/param.h"

=== modified file 'source4/torture/ldap/ldap_sort.c'
--- old/source4/torture/ldap/ldap_sort.c	2011-01-25 09:22:16 +0000
+++ new/source4/torture/ldap/ldap_sort.c	2011-02-27 16:58:37 +0000
@@ -22,8 +22,8 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "ldb_wrap.h"
 #include "param/param.h"
 #include "lib/cmdline/popt_common.h" 

=== modified file 'source4/torture/ldb/ldb.c'
--- old/source4/torture/ldb/ldb.c	2011-01-25 09:22:16 +0000
+++ new/source4/torture/ldb/ldb.c	2011-02-27 16:58:43 +0000
@@ -21,8 +21,8 @@
 
 #include "includes.h"
 #include "lib/events/events.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "lib/ldb-samba/ldif_handlers.h"
 #include "ldb_wrap.h"
 #include "dsdb/samdb/samdb.h"

=== modified file 'source4/torture/local/dbspeed.c'
--- old/source4/torture/local/dbspeed.c	2010-10-15 01:32:48 +0000
+++ new/source4/torture/local/dbspeed.c	2011-02-27 16:58:49 +0000
@@ -22,8 +22,8 @@
 #include "includes.h"
 #include "system/filesys.h"
 #include "../tdb/include/tdb.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "ldb_wrap.h"
 #include "lib/tdb_wrap.h"
 #include "torture/smbtorture.h"

=== modified file 'source4/torture/rpc/drsuapi_cracknames.c'
--- old/source4/torture/rpc/drsuapi_cracknames.c	2010-10-15 01:32:48 +0000
+++ new/source4/torture/rpc/drsuapi_cracknames.c	2011-02-27 16:58:53 +0000
@@ -24,7 +24,7 @@
 #include "includes.h"
 #include "librpc/gen_ndr/ndr_drsuapi_c.h"
 #include "torture/rpc/torture_rpc.h"
-#include "ldb/include/ldb.h"
+#include <ldb.h>
 #include "libcli/security/security.h"
 
 struct DsCrackNamesPrivate {

=== modified file 'source4/torture/rpc/netlogon.c'
--- old/source4/torture/rpc/netlogon.c	2011-01-25 09:22:16 +0000
+++ new/source4/torture/rpc/netlogon.c	2011-02-27 16:58:58 +0000
@@ -32,7 +32,7 @@
 #include "librpc/gen_ndr/ndr_lsa_c.h"
 #include "param/param.h"
 #include "libcli/security/security.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "lib/util/util_ldb.h"
 #include "ldb_wrap.h"
 #include "lib/replace/system/network.h"

=== modified file 'source4/utils/ntlm_auth.c'
--- old/source4/utils/ntlm_auth.c	2010-09-08 03:30:03 +0000
+++ new/source4/utils/ntlm_auth.c	2011-02-27 16:59:00 +0000
@@ -24,7 +24,7 @@
 #include "includes.h"
 #include "system/filesys.h"
 #include "lib/cmdline/popt_common.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "auth/credentials/credentials.h"
 #include "auth/gensec/gensec.h"
 #include "auth/auth.h"

=== modified file 'source4/winbind/idmap.c'
--- old/source4/winbind/idmap.c	2010-10-15 01:32:48 +0000
+++ new/source4/winbind/idmap.c	2011-02-27 16:59:03 +0000
@@ -22,7 +22,7 @@
 #include "includes.h"
 #include "auth/auth.h"
 #include "librpc/gen_ndr/ndr_security.h"
-#include "lib/ldb/include/ldb.h"
+#include <ldb.h>
 #include "ldb_wrap.h"
 #include "param/param.h"
 #include "winbind/idmap.h"

=== modified file 'source4/wrepl_server/wrepl_in_call.c'
--- old/source4/wrepl_server/wrepl_in_call.c	2010-09-08 03:30:03 +0000
+++ new/source4/wrepl_server/wrepl_in_call.c	2011-02-27 16:59:10 +0000
@@ -28,8 +28,8 @@
 #include "wrepl_server/wrepl_server.h"
 #include "libcli/composite/composite.h"
 #include "nbt_server/wins/winsdb.h"
-#include "lib/ldb/include/ldb.h"
-#include "lib/ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "system/time.h"
 #include "lib/util/tsort.h"
 #include "param/param.h"

=== modified file 'source4/wrepl_server/wrepl_scavenging.c'
--- old/source4/wrepl_server/wrepl_scavenging.c	2010-09-08 03:30:03 +0000
+++ new/source4/wrepl_server/wrepl_scavenging.c	2011-02-27 16:59:20 +0000
@@ -23,8 +23,8 @@
 #include "librpc/gen_ndr/ndr_winsrepl.h"
 #include "wrepl_server/wrepl_server.h"
 #include "nbt_server/wins/winsdb.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "system/time.h"
 #include "smbd/service_task.h"
 #include "lib/messaging/irpc.h"

=== modified file 'source4/wrepl_server/wrepl_server.c'
--- old/source4/wrepl_server/wrepl_server.c	2010-09-08 03:30:03 +0000
+++ new/source4/wrepl_server/wrepl_server.c	2011-02-27 16:59:26 +0000
@@ -27,8 +27,8 @@
 #include "librpc/gen_ndr/winsrepl.h"
 #include "wrepl_server/wrepl_server.h"
 #include "nbt_server/wins/winsdb.h"
-#include "ldb/include/ldb.h"
-#include "ldb/include/ldb_errors.h"
+#include <ldb.h>
+#include <ldb_errors.h>
 #include "auth/auth.h"
 #include "ldb_wrap.h"
 #include "param/param.h"

